.github/workflows/release.yml

```diff
@@ -122,6 +119,35 @@ jobs:
                 "${UPLOAD_URL}?name=$(basename ${FILE})";
             done
           done
+      - name: Create VERSION.txt
+        run: |
+          echo "$VERSION" > VERSION.txt
+      - name: Upload to artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: version
+          path: VERSION.txt
+  dispatch:
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download version
+        uses: actions/download-artifact@v3
+        with:
+          name: version
+      - name: Get Version
+        run: |
+          echo "VERSION=$(cat VERSION.txt)" >> $GITHUB_ENV
+      - name: Get branch name
+        run: echo "branch=${GITHUB_REF#refs/heads/}" >> $GITHUB_OUTPUT
+        id: ref
```

Comment:
You don't appear to be using this value. Is it still needed?

Reply:
@mcdonnnj not needed. resolved with [acfccc9](https://github.com/cisagov/con-pca-tasks/pull/8/commits/acfccc9ae4a98f8d4512331dff61a8b0414da49e)